enhancements/ci-enhancement.md

```diff
@@ -0,0 +1,73 @@
+# Summary
+
+This enhancement proposal is to improve the overall performance and functionality of the operator-sdk CI and testing. There are a number of issues currently plaguing our system, some of which can be handled by switching to a higher tier of compute service and some of which will require serious revisions to our testing process.
+
+# Motivations
+
+The primary motivation for these enhancements is to improve developer productivity. Much time is lost in slow CI runs, inconsistent flakes and failures in CI, and a general difficulty in debugging said CI runs. 
```

Comment:
I would note that another big step to improving productivity would be to make it easy, it at least possible, to run specific tests or set of tests in a way that accurately mimics the CI environment

Reply:
You say that below but I still think it's a major motivation